native/acter/src/api/spaces.rs

```diff
@@ -48,18 +49,7 @@ use super::{
 pub struct Space {
     pub client: Client,
     pub(crate) inner: Room,
-    task_list_event_handle: Option<EventHandlerHandle>,
-    task_list_update_event_handle: Option<EventHandlerHandle>,
-    task_event_handle: Option<EventHandlerHandle>,
-    task_update_event_handle: Option<EventHandlerHandle>,
-    comment_event_handle: Option<EventHandlerHandle>,
-    comment_update_event_handle: Option<EventHandlerHandle>,
-    pin_event_handle: Option<EventHandlerHandle>,
-    pin_update_event_handle: Option<EventHandlerHandle>,
-    calendar_event_event_handle: Option<EventHandlerHandle>,
-    calendar_event_update_event_handle: Option<EventHandlerHandle>,
-    news_entry_event_handle: Option<EventHandlerHandle>,
-    news_entry_update_event_handle: Option<EventHandlerHandle>,
+    handles: Vec<EventHandlerHandle>,
```

Comment:
It looks critical change that you replaced individual event handle with handle vector.

Reply:
Why is that critical?